arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -46,7 +40,13 @@
             "schema": {
               "$ref": "#/definitions/WebService"
             }
-          }                
+          },                
```

Comment:
Infact I would suggest - resourceGroupName, webserviceName, bodyParameter, api-version and then subscriptionID as the parameter